tools/rna_tools/locarna/locarna.xml

```diff
@@ -1,92 +0,0 @@
-<tool id="locarna" name="locarna" version="0.1.0">
-    <description></description>
-    <requirements>
-    <requirement type="package" version="x.y.z">locarna</requirement>
```

Comment:
Hi @s-will, historically I am not precisely knowledgeable about the details of *special* way graphclust invokes locarna. The locarna_graphclust wrapper is a wrapper that invokes locarna reliability script and further post-processes the mlocarna tree to get best subtree and because of that I asked @eteriSokhoyan to append the name graphclust to avoid confusion. Do you think it would be possible to merge locarna_graphclust   with locarna_multiple wrapper?